tests/search/includes/classes/class-concurrency-limiter-helper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Automattic\VIP\Search;
+
+require_once __DIR__ . '/../../../../search/includes/classes/class-concurrency-limiter.php';
+
+// phpcs:disable Generic.CodeAnalysis.UselessOverridingMethod.Found
+
+class Concurrency_Limiter_Helper extends Concurrency_Limiter {
+	public function get_key(): int /* NOSONAR */ {
+		return parent::get_key();
+	}
```

Comment:
My personal preference is not to use Reflection.

Reply:
What does that imply for us non-php native folks? :)

Reply:
Nothing :-) I will kill this class in #2706 anyway, so please don't pay attention :-)

https://www.cprime.com/resources/blog/unit-testing-smells/